published/external/xdpapi.h

```diff
@@ -292,6 +297,10 @@ typedef struct _XDP_API_CLIENT {
     XDP_API_PROVIDER_BINDING_CONTEXT *XdpApiProviderContext;
 
     //
+    // Rundown protection for API calls.
+    //
+    PEX_RUNDOWN_REF_CACHE_AWARE RundownRef;
```

Comment:
FWIW, I don't really think these need to be cache-aware - a simple EX_RUNDOWN_REF should be sufficient.

Reply:
If I use `EX_RUNDOWN_REF` here, what should be used in `XDP_FILE_OBJECT_HEADER`?
I simply copy this cache aware reference to the one in `XDP_FILE_OBJECT_HEADER`. Was the way wrong?

Reply:
Which is correct
https://github.com/microsoft/xdp-for-windows/pull/719#discussion_r1803016084

Reply:
maybe fixed
https://github.com/microsoft/xdp-for-windows/pull/719/commits/78a101d3fb7b2f4b6b327079652b29c8db7ef8c3